### PR TITLE
RD-6637 Replace sorting buttons with a dropdown

### PR DESCRIPTION
**Pull request by qooban**
## Description

Moved sorting buttons (used for Deployment Inputs and Workflow Parameters) into small dropdown menu.

## Screenshots / Videos

https://user-images.githubusercontent.com/5202105/214015488-0302778f-32ed-48ae-911d-84a7b28d6c6f.mp4

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
1. Tested locally - works fine.
2. [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4382/) - only modified tests - failed due to environment issues
3. [Build Status](https://jenkins.cloudify.co/job/Stage-UI-System-Test/4390/) - only modified tests - PASSED

## Documentation
This area will be further changed in RD-6486, so no need to update docs right now.

### CODE REVIEW

File: app/widgets/common/inputs/SortOrderIcons.tsx [hunk 1]
```diff
@@ -33,16 +36,24 @@ export default function SortOrderIcons({ onChange, selected }: SortOrderIconsPro
...
-                    key={sortOrder}
-                    link
-                    size="large"
-                    color={selected === sortOrder ? 'blue' : undefined}
-                    onClick={() => onChange(sortOrder)}
-                    {...iconProps}
-                />
-            ))}
+            <Dropdown
+                trigger={<Icon name={sortOrderToIconPropsMap[selected].name} size="large" aria-label="Sort order" />}
```
Comment (kubama) [inline app/widgets/common/inputs/SortOrderIcons.tsx:40]: Shouldn't `aria-label`s be i18n-ized?

**Review by kubama — COMMENTED** [review]
I'm thinking about the styling. It's now a button opening a dropdown menu. Shouldn't the styling be in line with the buttons on the right? I mean the background or at least a border.

File: app/widgets/common/inputs/SortOrderIcons.tsx (hunk 1 above)
Comment (qooban) [reply app/widgets/common/inputs/SortOrderIcons.tsx:40]: I wasn't sure at first, but... Yeah, it probably should. Will add translation support for it.

**Comment by qooban** [comment]
> I'm thinking about the styling. It's now a button opening a dropdown menu. Shouldn't the styling be in line with the buttons on the right? I mean the background or at least a border.

Yes, it's part of the next ticket - RD-6486.

**Review by kubama — APPROVED** [review]

**Pull request merged**